File: easyPopup/src/main/java/org/fattili/easypopup/view/pop/RightPop.java
package org.fattili.easypopup.view.pop;

import android.app.Activity;
import android.view.Gravity;
import android.view.View;

import org.fattili.easypopup.view.base.BasePopView;

/**
 * 2021/2/22
 *
 * @author dengzhenli
 */
public abstract class RightPop extends BasePopView {

    public RightPop(Activity activity) {
        super(activity);
        setGravity(Gravity.RIGHT);
    }

    public RightPop(Activity activity,View view) {
        super(activity, view);
    }


    @Override
    public void initView(View view) {

    }

    @Override
    public void initData() {

    }


    @Override
    public void onPopDismiss() {

    }
}

File: easyPopup/src/main/java/org/fattili/easypopup/view/pop/RightPop.kt
package org.fattili.easypopup.view.pop

import android.app.Activity
import android.view.Gravity
import org.fattili.easypopup.R
import org.fattili.easypopup.view.EasyPop

/**
 * 2021/2/22
 * 右侧弹出窗
 * @author dengzhenli
 */
abstract class RightPop : EasyPop {
    constructor(activity: Activity) : super(activity) {
        setGravity(Gravity.RIGHT)
        setAnimationStyle(R.style.ep_common_pop_right_theme)
    }
}